src/test/java/reservation/ReservationTest.java

```diff
@@ -0,0 +1,33 @@
+package reservation;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import roomescape.reservation.Reservation;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+public class ReservationTest {
+    @DisplayName("예약 정보와 id가 같은지 확인 : True")
+    @Test
+    void isSameIdTrueTest() {
+        //given
+        Reservation reservation = new Reservation(1L,"test1", LocalDate.of(2025, 4, 19), LocalTime.of(12, 13));
```

Comment:
코드 전반적으로 띄어쓰기/개행 등 확인해주세요~

```suggestion
        Reservation reservation = new Reservation(1L, "test1", LocalDate.of(2025, 4, 19), LocalTime.of(12, 13));
```